helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -35,9 +35,13 @@ data:
       -Dshare.protocol={{ .Values.externalProtocol | default "http"}}
       -Dshare.host={{ .Values.externalHost | default (printf "%s-share" (include "content-services.shortname" .)) }}
       -Dshare.port={{ .Values.externalPort | default .Values.share.service.externalPort }}
-      -Dalfresco_user_store.adminpassword={{ .Values.repository.adminPassword | default "209c6174da490caeb422f3fa5a7ae634" }}
+      -Dalfresco_user_store.adminpassword={{ .Values.repository.adminPassword | default "209c6174da490caeb422f3fa5a7ae634" }} {{ if index .Values "alfresco-search" "enabled" }}
       -Dsolr.host={{ template "alfresco-search.host" . }}
       -Dsolr.port={{ template "alfresco-search.port" . }}
+      {{ end }} {{ if index .Values "alfresco-search" "external" }}
```

Comment:
If alfresco search is enabled and external is present, then solr.host is set twice, right?

Reply:
The external port and host are commented out by default I'm expecting to not be both enable at the same time , but very good point, I will fix this.

Reply:
Done :)